Live/LiveEventWithDVR/Program.cs

```diff
@@ -1,663 +1,562 @@
-                #endregion
-
-
-                #region CreateLiveOutput
-                // Create the Live Output - think of this as the "tape recorder for the live event". 
-                // Live outputs are optional, but are required if you want to archive the event to storage,
-                // use the asset for on-demand playback later, or if you want to enable cloud DVR time-shifting.
-                // We will use the asset created above for the "tape" to record to. 
-                string manifestName = "output";
-                Console.WriteLine($"Creating a live output named {liveOutputName}");
-                Console.WriteLine();
+                AssetName = assetName,
+                StreamingPolicyName = "Predefined_ClearStreamingOnly"
+            })
+            ).Value;
+        Console.WriteLine("To playback the archived on-demand asset, Use the following urls:");
+        manifests = BuildManifestPaths(scheme, hostname, archiveLocator.Data.StreamingLocatorId.ToString(), manifestName);
+        Console.WriteLine($"The HLS (MP4) manifest for the archived asset without a DVR filter is : {manifests[0]}");
+        Console.WriteLine("Open the following URL to playback the live stream in an HLS compliant player (HLS.js, Shaka, ExoPlayer) or directly in an iOS device");
+        Console.WriteLine($"{manifests[0]}");
+        Console.WriteLine();
+        Console.WriteLine($"The DASH manifest URL for the archived asset without a DVR filter  : {manifests[1]}");
+        Console.WriteLine("Open the following URL to playback the live stream from the LiveOutput in the Azure Media Player");
+        Console.WriteLine($"https://ampdemo.azureedge.net/?url={manifests[1]}&heuristicprofile=lowlatency");
+        Console.WriteLine();
+        Console.WriteLine("Continue experimenting with the stream until you are ready to finish.");
+        Console.WriteLine("Press enter to stop the LiveOutput...");
+        Console.Out.Flush();
+        ignoredInput = Console.ReadLine();
+
+        Console.WriteLine("Experiment with playback of the live archive showing the full asset duration with the filter removed from the Streaming Locator");
+        Console.WriteLine("Press enter to stop and cleanup the sample...");
+        Console.Out.Flush();
+        ignoredInput = Console.ReadLine();
+    }
+}
+catch (RequestFailedException e)
+{
+    Console.WriteLine("Hit ErrorResponseException");
+    Console.WriteLine($"{e.Message}");
+    Console.WriteLine();
+    Console.WriteLine("Exiting, cleanup may be necessary...");
+    Console.ReadLine();
+}
+finally
+{
+    Console.WriteLine("Cleaning up resources, stopping Live Event billing, and deleting live Event...");
+    Console.WriteLine("CRITICAL WARNING ($$$$) DON'T WASTE MONEY!: - Wait here for the All Clear - this takes a few minutes sometimes to clean up. DO NOT STOP DEBUGGER yet or you will leak billable resources!");
```

Comment:
This comment doesn't inspire confidence about the product.

Reply:
It was intended to make you freak out....
It's true, if you don't do the right thing here, you will leak billable resources.  Happy to change that, but it is something we must improve on in aPaaS with a disconnection timeout property to auto shut off stuff.